lib/apr_web/helpers/view_helper.ex

```diff
@@ -1,11 +1,21 @@
 defmodule Apr.ViewHelper do
+  @exchange_url Application.get_env(:apr, :exchange)[:url]
+
   def exchange_link(order_id) do
-    Application.get_env(:apr, :exchange)[:url] <> "/admin/orders/#{order_id}"
+    "#{@exchange_url}/admin/orders/#{order_id}"
   end
 
   def currency(amount, currency \\ :USD) do
     amount
     |> Money.new(currency)
     |> Money.to_string()
   end
+
+  def exchange_user_orders_link(user_id) do
+    "#{@exchange_url}/admin/orders?q%5Bbuyer_id_eq=#{user_id}"
```

Comment:
I bet there are helpers that handle encoding and avoid having to manually construct querystrings.

Reply:
good call! yes updated it with using `URI.encode`